mobile/src/home-view.js

```diff
@@ -114,8 +115,17 @@ class HomeView extends PureComponent {
         reportRef.on('child_added', data => {
           this.setState({ reports: [...this.state.reports, data.key] })
         })
+        this.hideLogInScreen = setTimeout(() => {
+          this.setState( {isLoggedIn: true})
+        }, 500)
```

Comment:
I'm wondering whether 500ms is a bit too long.  It's noticeable enough to make things feel slow.  If 200ms is enough, that will feel noticeably snappier.